src/api/app/models/decision.rb

```diff
@@ -1,10 +1,14 @@
 class Decision < ApplicationRecord
+  TYPES = %i[DecisionFavored DecisionCleared].freeze
+
   validates :reason, presence: true, length: { maximum: 65_535 }
+  validates :type, presence: true, length: { maximum: 255 }
```

Comment:
I don't think we need this validation, the `type` will be set by the STI system automatically whenever we instantiante an inherited class, and we shouldn't tinker with the contents...

Reply:
Alright